polkadot/node/subsystem-bench/src/subsystem-bench.rs

```diff
@@ -54,16 +54,41 @@ fn le_5000(s: &str) -> Result<usize, String> {
 	number_range(s, 0, 5000)
 }
 
+/// Supported test objectives
+#[derive(Debug, Clone, Parser, Serialize, Deserialize)]
+#[command(rename_all = "kebab-case")]
+pub enum TestObjective {
+	/// Benchmark availability recovery strategies.
+	DataAvailabilityRead(DataAvailabilityReadOptions),
+	/// Benchmark availability and bitfield distribution.
+	DataAvailabilityWrite,
+	/// Benchmark the approval-voting and approval-distribution subsystems.
+	ApprovalVoting(ApprovalsOptions),
+	Unimplemented,
```

Comment:
why do we have unimplemented?

Reply:
Can't answer the question. It's not a part of my PR after I rebased the branch. And It was not me who implemented it before.